contributing.md

```diff
@@ -30,7 +30,7 @@ There is a webpage that has been developed by awesome members of the community u
 
 * If you are fixing a bug or adding a new feature, please create a branch for it.
 
-* Make Pull requests as decriptive as possible.
+* Make Pull requests as descriptive as possible.
```

Comment:
Thanks for noticing that, but this change should be ported to a different PR

Reply:
Done